src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -177,6 +178,9 @@ class QueueIO[T <: Data](private val gen: T, val entries: Int) extends Bundle
   val deq = Flipped(DeqIO(gen))
   /** The current amount of data in the queue */
   val count = Output(UInt(log2Ceil(entries + 1).W))
+  /** Input to choose if the queue should flush entries (Optional IO for a flushable Queue)*/ 
```

Comment:
```suggestion
  /** When asserted, reset the enqueue and dequeue pointers, effectively flushing the queue (Optional IO for a flushable Queue)*/ 
```

I think it's important to clarify whether this is actually erasing the data or not, since we aren't making any guarantees about that which might have security implications

Reply:
Okay yes that makes sense thank you!